packages/app-extensions/src/rest/helpers.js

```diff
@@ -33,21 +34,10 @@ export function* fetchEntity(entityName, key, query, transformer = json => json)
  *
  * @param entityName {String} Name of the entity
  * @param key {String} key of the entity
+ * @param type {String} type of the display, default display if none passed
  */
-export function* fetchDisplay(entityName, key) {
-  const cachedDisplay = cache.get('display', `${entityName}.${key}`)
-  if (cachedDisplay) {
-    return cachedDisplay
-  }
-  const options = {
-    method: 'GET'
-  }
-
-  const resp = yield call(requestSaga, `entities/2.0/${entityName}/${key}/display`, options)
-  const display = resp.body.display
-  cache.add('display', `${entityName}.${key}`, display)
-
-  return display
+export function* fetchDisplay(entityName, key, type) {
+  return _get(yield fetchDisplays({[entityName]: [key]}, type), `${entityName}.${key}`, null)
```

Comment:
unnecessary return

Reply:
I don't think it is, because it's not an arrow function. At least it doesn't work when I remove it.